docker-compose.yml

```diff
@@ -35,3 +38,15 @@ services:
       - .env
     ports:
       - 27018:27018
+  db_seed:
+    image: node:17
```

Comment:
Use node 16 instead, this way we dont need to download two separate node images.